recipes-extended/onl/files/accton-as4610/0004-accton_as4610_cpld-add-of-device-match-table.patch

```diff
@@ -0,0 +1,52 @@
+
```

Comment:
This empty line is not needed.

Reply:
Indeed, got added when copypasting. Dropped it.